### PR TITLE
use installer-log-file everywhere possible

### PR DESCRIPTION
**Pull request by evgeni**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/assembly_installing-server-connected.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ Note that the {Project} installation script is based on Puppet, which means that
 ⁠
 To avoid this and determine which future changes apply, use the `--noop` argument when you run the installation script.
 This argument ensures that no actual changes are made.
-Potential changes are written to `/var/log/foreman-installer/{project-context}.log`.
+Potential changes are written to `{installer-log-file}`.
```
Comment (evgeni) [inline guides/common/assembly_installing-server-connected.adoc:28]: I wonder whether this is correct for Orcharino? `project-context` is defined as `orcharino` there, but in other places of the docs you use `--scenario katello`.

Comment (evgeni) [reply guides/common/assembly_installing-server-connected.adoc:28]: It def changes the rendering from `/var/log/foreman-installer/orcharhino.log` to `/var/log/foreman-installer/foreman.log`, and I bet the later is totally wrong, but is the former right? Or should it be `katello.log`?

cc @maximiliankolb

Comment (maximiliankolb) [reply guides/common/assembly_installing-server-connected.adoc:28]: We use both `/var/log/foreman-installer/foreman.log` and `/var/log/foreman-installer/katello.log` with much much more content in the Katello log... 🤷 

Thanks for adding an attribute for this! I think `katello.log` is the right call!

Comment (evgeni) [reply guides/common/assembly_installing-server-connected.adoc:28]: *how* do you manage to get both? The installer should be very vocal not to do such things ;)

Or do you have setups with non-content proxies, that are deployed with `foreman-installer --scenario foreman --no-enable-foreman`?

Comment (ekohl) [reply guides/common/assembly_installing-server-connected.adoc:28]: Well, if you have the `foreman` scenario present it does run the migrations and that generates logs IIRC, even if you only use the scenario `katello`.

File: guides/common/modules/proc_configuring-installation.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ ifdef::foreman-el,katello[]
 For example, on Red Hat-based operating systems, use a utility such as `tmux` or `screen`.
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-installation.adoc:20]: Unrelated, but `tmux` and `screen` are not Red Hat specific. The whole sections needs a look. Also, `screen` is no longer in EL8 so also the Satellite-specific parts need some attention.

**Comment by evgeni** [comment]
Yes, I had the same thoughts, but didn't want to blow the scope of the PR ;)

File: guides/common/assembly_installing-server-connected.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/assembly_installing-server-connected.adoc:28]: I was specifically checking orcharhino Server; not an orcharhino Proxy. To my knowledge, we use scenario `katello`.

**Pull request merged**